python/vast/utils/config.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+import dynaconf
+
+CONFIG_FILES = ["config.yaml", "config.yml"]
+
+
+def create(config_files=CONFIG_FILES):
+    return dynaconf.Dynaconf(
+        settings_files=config_files,
+        load_dotenv=True,
+        envvar_prefix="VAST",
```

Comment:
I think this may lead to problems with VAST also running. Should we consider a separate prefix, e.g., `VAST_PYTHON`?

Reply:
I would like to see the two be really one in the future, which is why I think `VAST` is better suited. If this causes clashes, let's think about it.